examples/platform/silabs/SiWx917/BUILD.gn

```diff
@@ -52,6 +52,7 @@ import("${silabs_common_plat_dir}/SiWx917/args.gni")
 assert(chip_enable_wifi)
 assert(use_SiWx917)
 import("${chip_root}/src/platform/silabs/SiWx917/args.gni")
+wisemcu_sdk_root = "${chip_root}/third_party/silabs/wisemcu-wifi-bt-sdk"
```

Comment:
why do you need this path here?

Reply:
We are using wisemcu_sdk_root  variables for few files

source_set("siwx917-matter-shell") {
  if (chip_build_libshell) {
    defines = [ "ENABLE_CHIP_SHELL" ]

    sources = [
      "${silabs_common_plat_dir}/matter_shell.cpp",
      "${wisemcu_sdk_root}/platforms/si91x/drivers/cmsis_driver/UDMA.c",
      "${wisemcu_sdk_root}/platforms/si91x/drivers/cmsis_driver/USART.c",
    ]